src/layout/topic.js

```diff
@@ -63,6 +64,11 @@ Topic.propTypes = {
 	title: PropTypes.string.isRequired,
 
 	/**
+	 * description: (partials)(5)
```

Comment:
`(partials)(x)` will be a node... so this will likely error out when running docs...